oauth2/src/main/java/com/okta/spring/boot/oauth/env/OktaOAuth2PropertiesMappingEnvironmentPostProcessor.java

```diff
@@ -123,7 +123,7 @@ public Object getProperty(String name) {
 
     private PropertySource oktaRedirectUriPropertySource(Environment environment) {
         Map<String, Object> properties = new HashMap<>();
-        properties.put("spring.security.oauth2.client.registration.okta.redirect-uri", "{baseUrl}${okta.oauth2.redirect-uri}");
+        properties.put("spring.security.oauth2.client.registration.okta.redirect-uri", "${baseUrl}${okta.oauth2.redirect-uri}");
```

Comment:
I forgot about this one.  IIRC, the `{baseUrl }` is valid syntax and gets processed for each request
(so you can serve multiple domains from the same application)